python_ta/transforms/type_inference_visitor.py

```diff
@@ -146,63 +146,49 @@ def visit_name(self, node):
     ##############################################################################
     # Operation nodes
     ##############################################################################
-    def visit_binop(self, node):
-        t1 = self.type_constraints.lookup_concrete(node.left.type_constraints.type)
-        t2 = self.type_constraints.lookup_concrete(node.right.type_constraints.type)
-        op_name = op_to_dunder_binary(node.op)
-
+    def _lookify_call(self, node, func_name, *args):
```

Comment:
It's not good to make up words in names. You can use something a bit more generic like `handle_call`.